dhall-to-cabal.cabal

```diff
@@ -94,13 +94,12 @@ library
         bytestring ^>=0.10,
         containers ^>=0.5,
         contravariant ^>=1.4,
-        dhall ^>=1.12.0,
+        dhall ^>=1.13.0,
```

Comment:
As @quasicomputational noted, this should be done in the `dhall-to-cabal.dhall` file. I also think we should support `1.12` as well, as we don't depend on the API of 1.13.

Reply:
Ok, with the new "majorVersions" syntax it can be expressed with `dhall ^>=1.12.0 || ^>=1.13.0`, no ?

Reply:
Anyway, after a second thought, does not [this change in `lib/DhallToCabal.hs`](https://github.com/dhall-lang/dhall-to-cabal/pull/49/commits/c91e61804c2fddbb8007ec54ec02ab27cf6201c5#diff-d1c1b583c62f369b4b84141da6c209c2R557) make it depends on 1.13 exclusively?